src/translator/sentence_ranges.h

```diff
@@ -151,6 +151,14 @@ struct AnnotatedText {
   /// Returns a string_view representing sentence corresponding to sentenceIdx.
   string_view sentence(size_t sentenceIdx) const;
 
+  /// Returns the string_view of the filler between the previous sentence and
+  /// sentence described by sentenceIdx
+  string_view pre(size_t sentenceIdx) const;
+
+  /// Returns the string_view of the filler after the sentence corresponding to
+  /// sentenceIdx and the next sentence.
+  string_view post(size_t sentenceIdx) const;
```

Comment:
@jerinphilip Do you need both `post` and `pre` methods? Based on my understanding, `pre(n) = post(n-1)`. Right? Or did I miss something?

Reply:
Any alternative ideas, 

1. I can have an API with `.filler(i)`, where `i = -1..N-1 or 0..N+1`, while the sentences range from `[0, N-1]`. Use this to update the algorithm. I don't know if it is sane to have -1 or N+1 and create confusion while reading code. To me I prefer having `pre(i)` where I know it's filler before and `post(i)` where I know it's filler after while reading outside, not having to look up documentation.
2. I can return a pair of `(pre, post)` as `surroundings(i)` for the `i`-th sentence. Whoever calls the function outside decides which one to use to build their algorithm (faithful translation here). This is neater for readability, imo if you want a single function.

Reply:
I think 0 to N+1 is perfectly sensible here.

Reply:
Changed with `gap(i)`.